documentation/commands.md

```diff
@@ -186,6 +204,24 @@ Partial Route Example: `WHITE.J209.ORF` or `FESKO..RUFUS..CHOIR` or `KSEA.MTN7.E
 
 _Syntax -_ `AAL123 route [route]`
 
+### Reroute
+_Aliases -_ `reroute`, `rr`
+
+_Information -_ This command allows you to wipe out the aircraft's current
+route, and assign a new route of your choosing. This is similar to the `route`
+command, but this command will allow you to *change the entire route*, while the
+other is meant for specifying a route to follow to join a later point in the
+aircraft's flight plan. Note that the route uses dot format:
+
+>Note: Input data needs to be provided with single dots connecting all procedurally-
+linked points (eg KSFO.OFFSH9.SXC or SGD.V87.MOVER), and all other points that will
+be simply a fix direct to another fix need to be connected with double-dots
+(eg HLI..SQS..BERRA..JAN..KJAN).
+
+Full Route Example: `KSEA.HAROB5.ERAVE.Q1.ETCHY..MLBEC.BDEGA2.KSFO`
+
+_Syntax -_ `AAL123 rr [route]`
+
 ### Say Route
```

Comment:
This say route command can be removed. No need to scratch it out-- it's not that well known so we can just straight delete it.